posts/2022-01-31-configuring-ocp-for-user-namespace.md

```diff
@@ -0,0 +1,198 @@
+---
+title: Configuring ocp for user namespace
+published_date: "2022-01-31 11:45:58 +0000"
+layout: default.liquid
+is_draft: false
+---
+Preparing an openshift cluster for using user namespaces involve
+several steps and hands-over. To simplify the process we are using
+some configurations at freeipa-kustomize that make easier that task.
+
+**Pre-requisites**:
+
+- A 4.9+ OpenShift cluster.
+- You are logged in the cluster and you have cluster-admin privileges.
+- runc and cri-o rpm patched packages. Credits and thanks to [Fraser Tweedale](https://frasertweedale.github.io/blog-redhat/).
+  - [runc-1.0.3-992.rhaos4.10.el8.x86_64.rpm](https://ftweedal.fedorapeople.org/runc-1.0.3-992.rhaos4.10.el8.x86_64.rpm).
+  - [cri-o-1.23.0-990.rhaos4.10.git8c7713a.el8.x86_64.rpm](https://ftweedal.fedorapeople.org/cri-o-1.23.0-990.rhaos4.10.git8c7713a.el8.x86_64.rpm).
```

Comment:
Mentioning the experimental CRI-O build is perhaps not the best option - because it seems that https://github.com/cri-o/cri-o/pull/5552 is not likely to be accepted.  You can assign the `anyuid` SCC or use the `map-to-root=true` workaround.

Reply:
Thanks! removing them from here.